CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @owners will be requested for
+# review when someone opens a pull request.
+# Order is important; the last matching pattern takes the most
+# precedence.
+*   @tulir
+
+# Korean
+ko/ @revi
```

Comment:
You have no newline at the EOF. Is this intended?

Reply:
I don't think it matters? Unless it causes GitHub-side syntax error I don't feel like editing one…

Reply:
It can be trivial, but it is problem in some ways.
First, it is not a POSIX-compliant text file. Text file is a collection of lines, and a line must end with a newline.
Next, when someone adds a new line at the end of the file, git will detect the last line as edited. The diff can be different from what it really is. You can see this happens on [my PR](https://github.com/maunium/dontasktoask.com/pull/76/commits/21c50abd22366fcd65c12ee9fddf9791c4e7073e#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5). I just added a new line, however, git marks the previous line as edited.

Reply:
Imagine I will add myself as a codeowner of `ja/`. I will add a new line:`ja/ @aiotter`. Then the previous line `ko/ @revi` will be marked as edited.